### PR TITLE
Gas measurement and a little optimization of CliffFace.

### PR DESCRIPTION
**Pull request by gititGoro**
Small test on measuring gas differences between CliffFaceFlan swap vs EYE swap. Results should be taken with a pinch of salt.

### CODE REVIEW

File: test/wargames/ropsten.test.ts [hunk 1]
```diff
@@ -389,6 +404,82 @@ describe("ropsten deployment", function () {
     expect(changeInFlan.toString()).to.equal(impliedFlan)
   })
 
+  describe("gas test", () => {
+
+
+    this.afterEach(async () => {
+      const hre = await import("hardhat");
+      const provider = hre.network.provider;
```
Comment (khazra) [inline test/wargames/ropsten.test.ts:412]: Is it necessary to import `HRE` dynamically after each test? I would assume that importing it once at the file top level and then calling `hre.network.provider.send('evm_automine', [false])` should work the same here and potentially would be more performant.

**Review by khazra — APPROVED** [review]

File: test/wargames/ropsten.test.ts [hunk 2]
```diff
@@ -389,6 +404,82 @@ describe("ropsten deployment", function () {
...
+    return amountOut
+  }
+
+  const getBehodler = async (getContract: <T extends Contract> (contractName: contractNames, factoryName?: string, libraries?: ethersLib) => Promise<T>): Promise<Types.Behodler> => {
+    const addressBalanceCheck = await getContract("AddressBalanceCheck")
+    const behodlerLib: ethersLib = {
+      AddressBalanceCheck: addressBalanceCheck.address
+    }
+    return await getContract("Behodler", "Behodler", behodlerLib) as Types.Behodler
+  }
```
Comment (khazra) [inline test/wargames/ropsten.test.ts:482]: Both `getAmountOut` and `getBehodler` functions don't seem to be used anywhere. Why do we need them?

File: test/wargames/ropsten.test.ts (hunk 1 above)
Comment (gititGoro) [reply test/wargames/ropsten.test.ts]: Added a commit below to clean this up.

File: test/wargames/ropsten.test.ts (hunk 2 above)
Comment (gititGoro) [reply test/wargames/ropsten.test.ts]: removed unused functions

**Pull request merged**